README.md

```diff
@@ -434,7 +434,7 @@ On MacOS
 - AppleClang 11.0
 
 On Windows
-- Visual Studio 2017
```

Comment:
If we're keeping VS2017 coverage, then add VS2019 to the list instead of replacing.

Reply:
I'd actually suggest saying 2019, but adding a note saying "we don't drop support until we do (see https://github.com/KhronosGroup/SPIRV-Tools/pull/4983). Since we mostly keep it alive until is breaks, and will drop without notice.

Reply:
I've set it to VS2017, VS2019, and VS2022. As long as we test each of these, we should mention them in the README.